pkg/kubelet/kubelet_pods.go

```diff
@@ -146,6 +147,11 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 
 			hostPath = filepath.Join(hostPath, mount.SubPath)
 
+			err = volumevalidation.ValidatePathNoBacksteps(hostPath)
```

Comment:
Have to validate hostPath and mount.SubPath prior to calling filepath.Join, since it cleans/collapses backsteps